web/src/user_card_popover.js

```diff
@@ -89,7 +89,7 @@ class PopoverMenu {
             return;
         }
 
-        const $items = $("li:not(.divider):visible a:visible", $popover);
+        const $items = $("[tabindex='0']", $popover).filter(":visible");
```

Comment:
How does this syntax work? That first selector seems like it would not be limited to the popover.

Reply:
The first argument is the selectors to search for, and the second argument is the context to search within. And then we filter those results into only those which are visible. So this translates to: "All tabbable elements within the given popover instance which are also not hidden."

Reply:
This is similar to what `get_popover_items_for_instance` function does in `popover_menus.ts`, minus the `a` tag selector since that was just a failsafe for some popovers which don't set the `a` tags to `tabindex` of 0.